src/language/nmodl.yaml

```diff
@@ -1152,15 +1152,15 @@
                             type: Expression
 
                   - LinEquation:
-                      brief: "TODO"
+                      brief: "One equation in a system of equations tha collectively form a LINEAR block."
                       nmodl: "~ "
                       members:
-                        - left_linxpression:
```

Comment:
`linxpression` will be the name of my startup for coffee related products!

Reply:
that's one trademark issue preemptively avoided.